### DISCUSSION TITLE
Only workers, no master node?

### DISCUSSION THREAD

**Discussion by Taapo** [vitobotta/hetzner-k3s#378] [Q&A]
Is it possible to create a pool with only worker nodes, and no master node?

I want to launch worker nodes from an existing vps, and push commands to the worker nodes using kubectl for a specific use case.

**Comment by vitobotta** [comment]
Please open discussions for questions rather than issues. The worker node pools have only worker nodes. The only masters are in the master node pool.

**Comment by Taapo** [comment]
Sorry, didn't see discussions activated.

`The only masters are in the master node pool.`

I get that, but that's not what I asked. I don't want any masters (if possible at all), but cannot create a cluster without a master, since I'll get an warning on creation ...

**Reply by Taapo** [reply]
Seems I'm misunderstanding Kubernetes, as it apparently needs a master after all. Apologies.
